### PR TITLE
Documents update

### PR DESCRIPTION
**Pull request by razo7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,41 +3,41 @@
...
+- Build and run or deploy from sources to a running or to be created Openshift/Kubernetes cluster.
 
 ### Deploy the latest version
 
 After every PR merge to master images were build and pushed to `quay.io`.
-For deployment of NMO using these images you need
+For deployment of NMO using these images you need:
 
-- a running Openshift cluster, or a Kubernetes cluster with OLM installed
+- a running Openshioperator-sdk run bundle quay.io/kubevirt/node-maintenance-operator-bundle:latestft cluster, or a Kubernetes cluster with OLM installed.
```
[the remark this answers is not in the document]
Comment (razo7) [reply README.md:23]: Can you tell why? It is only punctuation change in the docs

[the remark this answers is not in the document]
Comment (slintes) [reply README.md:23]: no...? There is more than just the added  `.` in the end

**Review by slintes — APPROVED** [review]
/lgtm
/approve
/retest

**Pull request merged**